src/fides/api/ops/schemas/saas/saas_config.py

```diff
@@ -369,10 +373,8 @@ def get_graph(self, secrets: Dict[str, Any]) -> Dataset:
 
             if fields:
                 grouped_inputs: Set[str] = set()
-                for read_request in read_requests:
-                    grouped_inputs = grouped_inputs.union(
-                        read_request.grouped_inputs or []
-                    )
+                if read_requests:
+                    grouped_inputs = set(read_requests[0].grouped_inputs or [])
```

Comment:
this change looks good, thanks for addressing. i'm gonna be annoying though and suggest you throw in a small comment here that makes it clear we've already validated that all `grouped_inputs` are equivalent and that's why we're doing this 😄

Reply:
So annoying! 🙄

Reply:
:|